up-and-running/auth/ldap.md

```diff
@@ -0,0 +1,86 @@
+# LDAP Authentication
+
+npm Enterprise supports most LDAP corporate directories, including Active Directory and OpenLDAP.
+
+It's important to note that, when using LDAP integration, your npm Enterprise instance(s) should typically be running on the same subnet as your LDAP servers.
+
+Note that you can also use LDAP integration for logging into the admin console (port `:8800`) as well. If you'd like to set that up, visit the `/create-password` page once you're logged in, choose the `LDAP`, and enter the settings just as you would below.
+
+## Configuration Fields
+
+### Server Settings
+
+#### LDAP Type
+
+_Required._ What type of LDAP does your corporate directory use?
+
+Options are `OpenLDAP`, `Active Directory`, and `Other`.
+
+#### Hostname
+
+_Required._ The hostname or domain name that represents your LDAP server.
+
+#### Port
+
+_Required._ The port that should be used when accessing the LDAP server.
+
+Default is `389`.
+
+#### Encryption Type
+
+_Required._ The type of encryption your LDAP provider supports.
+
+Options are `Plain`, `StartTLS`, and `LDAPS`.
+
+#### Search user
+
+_Required._ The user, in distinguished name (DN) format, that npm Enterprise should use to log into LDAP. When users attempt to authenticate against the private npm registry or website, npm Enterprise will use this user to query the corporate directory service. This should typically include at least one common name (CN) representing a read-only service or admin user.
```

Comment:
Great description, I feel like I understand more about this field now ... I feel like the help I've given people on this topic has always been slightly hand-wavy.